### PR TITLE
Runs in the browser.

### PR DESCRIPTION
**Pull request by pentacular**

### CODE REVIEW

File: algorithm/threejs/trianglesToThreejsDatasets.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  let index = map[key];
+  if (index !== undefined) {
+    return index;
+  }
+  map[key] = next;
+  update(point);
+  return next;
+};
+
+export const trianglesToThreejsDatasets = (options = {}, ...triangularGeometries) => {
```
Comment (BarbourSmith) [inline algorithm/threejs/trianglesToThreejsDatasets.js]: Awesome. This is exactly how I was thinking to do this

File: api/v1/writePdf.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { pathsToPdf } from '@jsxcad/algorithm-pdf';
+import { writeFileSync } from '@jsxcad/sys';
 
 export const writePdf = ({ path }, shape) => {
   const paths = shape.toPaths({});
-  // TODO: Need to abstract filesystem access so that it can work in a browser.
-  require('fs').writeFileSync(path, pathsToPdf({}, paths));
+  writeFileSync(path, paths, { translator: () => pathsToPdf({}, paths) });
```
Comment (BarbourSmith) [inline api/v1/writePdf.js]: 👍

File: app/v1/dist/index.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+    </style>
+  </head>
+  <body>
+    <div id="container"></div>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/three.js/87/three.min.js"></script>
+    <script type="text/javascript" src="https://cdn.rawgit.com/mrdoob/stats.js/master/build/stats.min.js"></script>
+    <script type="text/javascript" src="https://cdn.rawgit.com/mrdoob/three.js/master/examples/js/controls/TrackballControls.js"></script>
+    <script type="text/javascript" src="https://cdn.rawgit.com/dataarts/dat.gui/master/build/dat.gui.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/ami.js//0.0.20/ami.min.js"></script>
+    <script type="module">
```
Comment (BarbourSmith) [inline app/v1/dist/index.html]: Will the behavior of this script change with webpack? Like will this behavior be a part of the webpack bundle, or will it always be separate?

File: algorithm/threejs/trianglesToThreejsDatasets.js (hunk 1 above)
Comment (pentacular) [reply algorithm/threejs/trianglesToThreejsDatasets.js]: Cool. :)

File: app/v1/dist/index.html (hunk 3 above)
Comment (pentacular) [reply app/v1/dist/index.html]: I don't have any particular preference -- feel free to change it if there's some benefit.

**Pull request merged**